platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -96,16 +112,20 @@ class HangingProtocolService {
     this.studies = [];
     this.protocols = [];
     this.hpAlreadyApplied = [];
-    this.matchDetails = [];
+    this.viewportMatchDetails = [];
     // this.ProtocolEngine.reset()
   }
 
-  public getDisplaySetsMatchDetails() {
-    return this.displaySetMatchDetails;
-  }
-
-  public getState() {
-    return [this.matchDetails, this.hpAlreadyApplied];
+  public getMatchDetails(): {
+    viewportMatchDetails: ViewportMatchDetails[];
```

Comment:
Isn't this the same type one gets for the event, or else very similar?  If so, it should have it's own interface.